docker/README.md

```diff
@@ -3,46 +3,86 @@
 
 You can build a docker image based on this Dockerfile. This image will contain only an Orion Context Broker instance, exposing port `1026`. This requires that you have [docker](https://docs.docker.com/installation/) installed on your machine.
 
-If you just want to have an Orion Context Broker running as fast as possible jump to section [The Fastest Way](#the_fastest_way).
+If you just want to have an Orion Context Broker running as fast as possible jump to section *The Fastest Way*.
 
-If you want to know what and how we are doing things with the container step by step you can  go ahead and read the [Build](#build_the_image) and [Run](#run_the_container) sections.
+If you want to know what and how we are doing things with the container step by step you can go ahead and read the build and run sections.
 
```

Comment:
If you want to know what is behind the scenes of our container you can go ahead and read the build and run sections